plugins/ipam/host-local/backend/disk/backend.go

```diff
@@ -55,7 +57,9 @@ func New(network, dataDir string) (*Store, error) {
 	return &Store{lk, dir}, nil
 }
 
-func (s *Store) Reserve(id string, ip net.IP, rangeID string) (bool, error) {
+// Reserve attempts to claim an ip in the name of a container with the given id.
```

Comment:
I'd avoid "name" here, since containers do have names, and we are using ID+interface as the key.

How about "claim an ip for a given container interface" ?